pkg/cnfs/v1beta1/types.go

```diff
@@ -82,6 +82,7 @@ type Parameters struct {
 	ACLType              string  `json:"aclType,omitempty"`
 	EnableTrashCan       string  `json:"enableTrashCan,omitempty"`
 	TrashCanReservedDays string  `json:"trashCanReservedDays,omitempty"`
+	AllowIncrease        string  `json:"allowIncrease,omitempty"`
```

Comment:
These are parameters for "cnfs" CSI driver, IMO it's not related to Disk CSI driver.

Reply:
Will remove. Thanks!